app/bundles/CoreBundle/Entity/VariantEntityTrait.php

```diff
@@ -4,17 +4,18 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping\Entity;
 use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
 
 trait VariantEntityTrait
 {
     /**
-     * @var ArrayCollection
+     * @var mixed
```

Comment:
This won't work? Similar to other changes in this file
```suggestion
     * @var ArrayCollection< VariantEntityInterface >
```